tools/es/build.js

```diff
@@ -12,16 +12,28 @@ export default function BuildES() {
     .then(() => buildBabel(srcRoot, esRoot, {
       babelrc: false,
       presets: [
-        ['es2015', { loose: true, modules: false }],
-        'stage-1',
-        'react',
+        ['env', {
+          'loose': true,
+          'targets': {
```

Comment:
the diifference here is in this one we wnat `"modules": false"` so they aren't compiled